htdocs/install/mysql/migration/12.0.0-13.0.0.sql

```diff
@@ -555,3 +555,6 @@ ALTER TABLE llx_product_fournisseur_price ADD COLUMN packaging varchar(64);
 
 ALTER TABLE llx_projet ADD COLUMN fk_opp_status_end integer DEFAULT NULL;
 
+ALTER TABLE llx_mrp_production ADD COLUMN fk_soc integer NOT NULL;
```

Comment:
What is the goal of this new column ? It seems not used by the code.

Reply:
Sorry, my analysisis of the problem was wrong. I didn't see "llx_mrp_mo" table. Here the right fix.